gulpfile.js

```diff
@@ -84,21 +95,21 @@ function javascript() {
       ),
     )
     .pipe($.if(!PRODUCTION, $.sourcemaps.write()))
-    .pipe(gulp.dest('dist'));
+    .pipe(gulp.dest(`${PATHS.ASSETS}/js`));
 }
 
 function server(done) {
   browser.init(
     {
-      server: './',
+      server: PATHS.DIST,
```

Comment:
Change server config so it is pointing to the `dist` folder when I run `npm start`